### DISCUSSION TITLE
Enable local.rules in an offline standalone installation

### DISCUSSION THREAD

**Discussion by bamaboy5692** [Security-Onion-Solutions/securityonion#3031] [Unsupported Versions]
How can I enable local.rules in an offline standalone install of security onion 2.3? I've tried to change directory into local.rules it sees it but says it doesn't exist.

**Comment by TOoSmOotH** [comment]
Did you select Airgap during install?

**Reply by bamaboy5692** [reply]
I did

**Reply by bamaboy5692** [reply]
Is this why I can't reach it in the directory?

**Reply by TOoSmOotH** [reply]
It's not a directory its a file. You need to paste your local rules into the file /opt/so/saltstack/local/salt/idstools/local.rules
